src/swagger.js

```diff
@@ -0,0 +1,41 @@
+import { signUp } from "./docs-data"
+export const swaggerDocument = {
+    openapi: '3.0.1',
+    info: {
+        version: '1.0.0',
+        title: 'Destructors APIs Document',
+        description: 'This a simple api for mastering about back-end which will do an ecommerce server',
```

Comment:
we're lacking Schemes here?

Reply:
You can checkout here https://portifolio-website.up.railway.app/api-docs/ what is Schemes!